### ISSUE TITLE
Uncaught EvalError: Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an al...

### ISSUE THREAD

**Issue by team6612** [olmokramer/atom-coffee-autocompile#4]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.0.19
**System**: Unknown Windows Version
**Thrown From**: [coffee-autocompile](https://github.com/olmokramer/atom-coffee-autocompile) package, v0.3.2

### Stack Trace

Uncaught EvalError: Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an allowed source of script in the following Content Security Policy directive: "script-src 'self'".

```
At C:\Users\daniel\.atom\packages\coffee-autocompile\node_modules\uglify-js\tools\node.js:24

EvalError: Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an allowed source of script in the following Content Security Policy directive: "script-src 'self'".

  at Function (native)
  at Object.<anonymous> (C:\Users\daniel\.atom\packages\coffee-autocompile\node_modules\uglify-js\tools\node.js:24:4)
  at Module._compile (module.js:428:26)
  at Object.defineProperty.value [as .js] (C:\Users\daniel\AppData\Local\atom\app-1.0.19\resources\app.asar\src\compile-cache.js:169:21)
  at Module.load (module.js:353:32)
  at Function.Module._load (module.js:308:12)
  at Module.require (module.js:363:17)
  at require (module.js:382:17)
  at Object.<anonymous> (C:\Users\daniel\.atom\packages\coffee-autocompile\lib\coffee-autocompile.js:30:17)
  at Module._compile (module.js:428:26)
  at Object.defineProperty.value [as .js] (C:\Users\daniel\AppData\Local\atom\app-1.0.19\resources\app.asar\src\compile-cache.js:169:21)
  at Module.load (module.js:353:32)
  at Function.Module._load (module.js:308:12)
  at Module.require (module.js:363:17)
  at require (module.js:382:17)
  at Package.module.exports.Package.requireMainModule (C:\Users\daniel\AppData\Local\atom\app-1.0.19\resources\app.asar\src\package.js:761:34)
  at Package.module.exports.Package.activateConfig (C:\Users\daniel\AppData\Local\atom\app-1.0.19\resources\app.asar\src\package.js:263:14)
  at PackageDetailView.module.exports.PackageDetailView.activateConfig (C:\Users\daniel\AppData\Local\atom\app-1.0.19\resources\app.asar\node_modules\settings-view\lib\package-detail-view.js:212:26)
  at PackageDetailView.module.exports.PackageDetailView.completeInitialzation (C:\Users\daniel\AppData\Local\atom\app-1.0.19\resources\app.asar\node_modules\settings-view\lib\package-detail-view.js:156:12)
  at PackageDetailView.module.exports.PackageDetailView.loadPackage (C:\Users\daniel\AppData\Local\atom\app-1.0.19\resources\app.asar\node_modules\settings-view\lib\package-detail-view.js:168:21)
  at C:\Users\daniel\AppData\Local\atom\app-1.0.19\resources\app.asar\node_modules\settings-view\lib\package-detail-view.js:298:17
  at Emitter.module.exports.Emitter.emit (C:\Users\daniel\AppData\Local\atom\app-1.0.19\resources\app.asar\node_modules\event-kit\lib\emitter.js:82:11)
  at PackageManager.module.exports.PackageManager.emitPackageEvent (C:\Users\daniel\AppData\Local\atom\app-1.0.19\resources\app.asar\node_modules\settings-view\lib\package-manager.js:588:27)
  at C:\Users\daniel\AppData\Local\atom\app-1.0.19\resources\app.asar\node_modules\settings-view\lib\package-manager.js:379:26
  at exit (C:\Users\daniel\AppData\Local\atom\app-1.0.19\resources\app.asar\node_modules\settings-view\lib\package-manager.js:60:16)
  at triggerExitCallback (C:\Users\daniel\AppData\Local\atom\app-1.0.19\resources\app.asar\src\buffered-process.js:213:47)
  at ChildProcess.<anonymous> (C:\Users\daniel\AppData\Local\atom\app-1.0.19\resources\app.asar\src\buffered-process.js:235:18)
  at emitTwo (events.js:87:13)
  at ChildProcess.emit (events.js:172:7)
  at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)

```

### Commands

```
     -4:53.4.0 core:save (atom-text-editor.editor.is-focused)
     -4:51.7.0 core:backspace (atom-text-editor.editor.is-focused)
     -4:51.2.0 core:save (atom-text-editor.editor.is-focused)
     -4:30.6.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -4:16 core:save (ol.tree-view.list-tree.has-collapsable-children.focusable-panel.full-menu)
     -4:06.7.0 tree-view:remove (ol.tree-view.list-tree.has-collapsable-children.focusable-panel.full-menu)
     -2:09.3.0 core:move-left (atom-text-editor.editor.is-focused)
     -2:07.3.0 core:save (atom-text-editor.editor.is-focused)
     -1:53.1.0 tree-view:remove (ol.tree-view.list-tree.has-collapsable-children.focusable-panel.multi-select)
     -1:36.6.0 core:paste (atom-text-editor.editor.is-focused)
     -1:35.7.0 core:save (atom-text-editor.editor.is-focused)
     -1:29.9.0 core:undo (atom-text-editor.editor.is-focused)
     -1:25 core:save (atom-text-editor.editor.is-focused)
     -1:08.0 tree-view:remove (ol.tree-view.list-tree.has-collapsable-children.focusable-panel.multi-select)
     -1:01.5.0 core:save (atom-text-editor.editor.is-focused)
     -0:29.4.0 core:confirm (atom-text-editor.editor.mini.is-focused)
```

### Config

```json
{
  "core": {}
}
```

### Installed Packages

```coffee
# User
angularjs, v0.3.3
atom-beautify, v0.28.14
atom-html-preview, v0.1.11
atom-jade, v0.3.0
autocomplete-sass, v0.1.0
color-picker, v2.0.13
emmet, v2.3.13
open-recent, v2.3.0
remote-edit, v1.8.21
sass-autocompile, v0.10.4
save-session, v0.15.7
sync-settings, v0.6.0

# Dev
No dev packages
```

**Comment by olmokramer** [comment]
I think this is a duplicate of https://github.com/olmokramer/atom-coffee-autocompile/issues/3

**Comment by olmokramer** [comment]
Just published v0.3.3 that should fix this issue. Please let me know if it works

**Comment by olmokramer** [comment]
Oops, I forgot to include a dependency in `package.json`. v0.3.5 should fix this.
